.github/workflows/release.yml

```diff
@@ -156,3 +156,9 @@ jobs:
           files: |
             phylum-*.zip
             phylum-*.zip.minisig
+
+  Deploy-Docs:
+    name: Deploy Docs
+    needs: Release
+    uses: ./.github/workflows/docs.yml
+    secrets: inherit
```

Comment:
Are any secrets from this caller workflow actually needed/used in the called workflow? Is this line just pre-emptive, in case there are secrets to pass in the future?

Reply:
The docs workflow needs `secrets.README_API` and it wasn't clear from the docs if that would be set without doing it this way.